.github/workflows/run-tests.yml

```diff
@@ -21,6 +21,17 @@ jobs:
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         run: hatch run test:run-report
+      - name: Run CLIs against website
```

Comment:
could we run this as a cron job as well? i ask because often what happens is that there is an issue in how a user fills out an issue. so they do something wierd (text wise) and it breaks the build. i was thikning if we ran this every week or twice a week that would catch those issues before we go to update contributors (i think?) i'm not sure if that thought process makes sense or not...

 but in this case i definitely broke something via a pr and this will fix that instance!

Reply:
done!